stout/event-loop.h

```diff
@@ -177,6 +177,51 @@ class EventLoop : public Scheduler {
     uv_signal_t* handle_ = nullptr;
   };
 
+  class Timer {
+   public:
+    Timer(EventLoop& loop) {
+      handle_ = new uv_timer_t();
+      CHECK_EQ(
+          uv_timer_init(loop, handle_),
+          0);
+    }
+
+    Timer(const Timer& that) = delete;
+
+    Timer(Timer&& that) {
+      handle_ = that.handle_;
+      that.handle_ = nullptr;
+    }
+
+    ~Timer() {
+      if (handle_) {
+        if (uv_is_active(reinterpret_cast<uv_handle_t*>(handle_))) {
+          uv_timer_stop(handle_);
+        }
+
+        uv_close(
+            reinterpret_cast<uv_handle_t*>(handle_),
+            EventLoop::CloseCallback);
+      }
+    }
+
+    // Adaptors to libuv functions.
+    uv_timer_t* handle() {
+      CHECK(handle_);
```

Comment:
As on the other PR, I suggest:
```suggestion
      CHECK_NOTNULL(handle_);
```